src/lib/state/reducers/files.js

```diff
@@ -32,15 +32,17 @@ const initialState = {};
 
 const fileReducer = (state = initialState, action) => {
   let newState;
+  // Filename needs to be normalised due to encoding differences between client and server.
+  let remoteFileName = action.payload?.filename ? action.payload.filename.normalize() : '';
```

Comment:
I think this can be reduced to 
```suggestion
  let remoteFileName = action.payload?.filename?.normalize() ?? '';
```